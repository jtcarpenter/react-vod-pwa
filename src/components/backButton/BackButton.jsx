import React from 'react';
import PlayerCtrl from 'components/playerCtrl/PlayerCtrl.jsx';
import * as copy from 'constants/copy';

export function BackButton({handleBack}) {

    return (
        <PlayerCtrl
            type="button"
            icon={copy.ICON_BACK}
            borderRadius="10px"
            onClick={handleBack}
        >
        </PlayerCtrl>
    );
}

export default BackButton;