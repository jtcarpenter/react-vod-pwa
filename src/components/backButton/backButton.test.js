import React from 'react';
import {shallow} from 'enzyme';
import renderer from 'react-test-renderer';
import {BackButton} from 'components/backButton/BackButton';

const mockBackButton = (
    <BackButton />
)

describe('BackButton', () => {
    it('should render', () => {
        const wrapper = shallow(mockBackButton);
        expect(wrapper.exists()).toBe(true);
    });

    it('should display correctly', () => {
        const backButton = renderer.create(mockBackButton);
        expect(backButton.toJSON()).toMatchSnapshot();
    });
});
