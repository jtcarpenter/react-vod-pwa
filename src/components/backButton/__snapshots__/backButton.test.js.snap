// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BackButton should display correctly 1`] = `
<button
  className="sc-bdVaJa cUeAZT"
  icon="\\\\ea40"
  onClick={undefined}
  type="button"
/>
`;
